apps/web/src/app/(private)/settings/page.tsx

```diff
@@ -2,9 +2,22 @@
 
 import { Settings } from '@latitude-data/web-ui'
 import useCurrentWorkspace from '$/stores/currentWorkspace'
+import useProviderApiKeys from '$/stores/providerApiKeys'
+import useUsers from '$/stores/users'
 
 export default function SettingsPage() {
   const { data, update } = useCurrentWorkspace()
+  const { data: apiKeys, create, destroy } = useProviderApiKeys()
+  const { data: users } = useUsers()
 
-  return <Settings workspace={data!} updateWorkspace={update} />
+  return (
+    <Settings
+      users={users}
+      workspace={data}
```

Comment:
Workspace is in the context

Reply:
Long story short I can't make it work with sessions without some serious refactor I think we have to rethink the whole web-ui/web divide. Not being able to use stores in web-ui components is a massive pita.